validator_test.sh

```diff
@@ -139,15 +138,15 @@ TX_QUERY_REG_TX_HASH=$(echo $RES | jq --raw-output '.txhash')
 echo "TX Query ID: $TX_QUERY_ID, TX hash: $TX_QUERY_REG_TX_HASH"
 
 echo ""
-echo -n "Checking that deposits have been deducted completely from contract balance… "
+echo -n "Checking that the deposit have been deducted completely from contract balance… "
+
 RES=$(${BIN} query bank balances ${CONTRACT_ADDRESS} --output json --node ${NODE_URL})
-BALANCE=$(echo $RES | jq --raw-output '[.balances[] | select(.denom == "untrn")][0].amount')
-if [[ $BALANCE -eq 20000 ]]; then
+BALANCE_AFTER_ICQ_REG=$(echo $RES | jq --raw-output '[.balances[] | select(.denom == "untrn")][0].amount')
```

Comment:
I think we can add small delay before querying for balance update (5-7 seconds).

Reply:
is it needed given the --broadcast-mode=block used in the above txs?

Reply:
sometimes it fails but still sits in the the mempool and waiting to be included into block.

Reply:
It will add more stability to the script imho.